R-package/R/utils.R

```diff
@@ -25,20 +20,19 @@ lgb.encode.char <- function(arr, len) {
 }
 
 lgb.call <- function(fun_name, ret, ...) {
-  
   # Set call state to a zero value
-  call_state <- 0L
+  call_state <- as.integer(0L)
   
   # Check for a ret call
   if (!is.null(ret)) {
     call_state <- .Call(fun_name, ..., ret, call_state, PACKAGE = "lib_lightgbm") # Call with ret
   } else {
     call_state <- .Call(fun_name, ..., call_state, PACKAGE = "lib_lightgbm") # Call without ret
   }
-  
+  call_state <- as.integer(call_state)
   # Check for call state value post call
   if (call_state != 0L) {
-    
```

Comment:
@guolinke Is it this line?

https://github.com/Microsoft/LightGBM/blob/master/R-package/R/lgb.Booster.R#L96 (your link is pointing to a dead page)